test/e2e/dra/dra.go

```diff
@@ -832,6 +832,8 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 
 			// Eventually the quota status should consider the existing claim.
 			gomega.Eventually(ctx, framework.GetObject(f.ClientSet.CoreV1().ResourceQuotas(quota.Namespace).Get, quota.Name, metav1.GetOptions{})).
+				WithPolling(framework.Poll).
+				WithTimeout(framework.PollShortTimeout).
```

Comment:
This doesn't help:
- framework.Poll is already the default (https://github.com/kubernetes/kubernetes/blob/5f594f42159f2ee84f805d382638f77dd5967177/test/e2e/framework/test_context.go#L509)
- without an explicit timeout, it waits forever (should use the default, but doesn't yet)

Reply:
This isn't the line where it fails either:

https://github.com/kubernetes/kubernetes/blob/5f594f42159f2ee84f805d382638f77dd5967177/test/e2e/dra/dra.go#L845

Reply:
I left some comments in the issue about this.

Reply:
> without an explicit timeout, it waits forever (should use the default, but doesn't yet)

@pohly I change the goal of this PR to add a timeout.

Reply:
Can we use `f.Timeouts.PodStart` instead of the constant https://github.com/carlory/kubernetes/blob/1e1eda6481ebb4555b77b15a0152810f66439951/test/e2e/dra/dra.go#L54?

Reply:
There's no need to add an explicit timeout when the default timeout will soon work (https://github.com/onsi/gomega/releases/tag/v1.35.0).

Reply:
/close